sympy/physics/mechanics/tests/test_eombase.py

```diff
@@ -0,0 +1,30 @@
+import sympy.physics.mechanics.eombase as eombase
+
+# Determine what inputs should be used
+EOM = eombase.EOMBase(frame=None)
+
+# Should contain access to all the property attributes contained in existing EOM
+# generators
+
+Thing = "Place holder until a correct output is chosen"
+
+assert EOM.auxiliary_eqs == Thing
+assert EOM.bodylist == Thing
+assert EOM.mass_matrix == Thing
+assert EOM.mass_matrix_full == Thing
+assert EOM.forcelist == Thing
+assert EOM.forcing == Thing
+assert EOM.forcing_full == Thing
+assert EOM.q == Thing
+assert EOM.u == Thing
```

Comment:
Here is what I see as the essentials to describe the EoMs of a multibody system:

Option 1

1. The system states, `x`.
2. The first order ordinary differential equation, i.e the right hand side of `x' = f`.

For a multibody system it may be worth knowing which entries of `f` are associated with the generalized speeds and which are associated with the generalized coordinates.

Option 2
1. The system states, `x`.
2. The implicit first order ordinary differential equation in the form `M x' = F`.

Lots more...will add later.